dataproc_jupyter_plugin/controllers/dagController.py

```diff
@@ -28,12 +28,12 @@
 
 class DagListController(APIHandler):
     @tornado.web.authenticated
-    def get(self):
+    async def get(self):
         try:
             dag = DagListService()
             composer_name = self.get_argument("composer")
-            credentials = GetCachedCredentials.get_cached_credentials(self.log)
-            dag_list = dag.list_jobs(credentials, composer_name, TAGS, self.log)
+            gcp_credentials = await credentials.get_cached()
+            dag_list = dag.list_jobs(gcp_credentials, composer_name, TAGS, self.log)
```

Comment:
Change this to:

```
composer_name = self.get_argument("composer")
client = airflow.Client(await credentials.get_cached(), self.log)
dag_list = await client.list_dags(composer_name, TAGS)
```

... and make similar changes to all of the other handlers in this file.